### DISCUSSION TITLE
mpif90 not finding compiler

### DISCUSSION THREAD

**Discussion by brianthomas-washburn** [alphaparrot/ExoPlaSim#34] [Q&A] [answered]
Hi all - this seems like something I should be able to solve, but haven't been able to yet.

When trying to compile, I get this message:
"The Open MPI wrapper compiler was unable to find the specified compiler
x86_64-conda-linux-gnu-gfortran in your PATH."

It seems the compiler should just be "gfortran" not "x86_64-conda-linux-gnu-gfortran" which might explain why it's not finding it. 

Did the exoplasim configure script set this or should I look elsewhere?

Thanks in advance!

**Comment by alphaparrot** [comment, marked as answer]
Hi, you may not have the conda environment that includes this gfortran
compiler activated—ExoPlaSim's configuration script found this gfortran on
the path when it ran, and if the environment isn't activated when running
ExoPlaSim, it won't see the same PATH.

**Reply by brianthomas-washburn** [reply]
Thank you, I'm new-ish to conda.  I did a conda install of gfortran and that fixed the issue.  It now compiles and is running (successfully ran all the tutorial notebook).  Much appreciated!
